rasa/core/evaluation/marker_tracker_loader.py

```diff
@@ -96,4 +96,4 @@ def load(self) -> Iterator[Optional[DialogueStateTracker]]:
 
         keys = self.strategy(stored_keys, self.count)
         for sender in keys:
-            yield self.tracker_store.retrieve(sender)
+            yield self.tracker_store.retrieve_full_tracker(sender)
```

Comment:
Ah, this is the only thing from the first version which we didn't check / adapt to work with multiple sessions. Looks like feature complete now 🥳